src/shared/config/config.env.ts

```diff
@@ -93,4 +95,10 @@ export default (): Config => ({
         KEYCLOAK_GROUP: process.env['VIDIS_KEYCLOAK_GROUP'],
         KEYCLOAK_ROLE: process.env['VIDIS_KEYCLOAK_ROLE'],
     },
+    IMPORT: {
+        PASSPHRASE_SECRET: process.env['PASSPHRASE_SECRET'],
+        IMPORT_FILE_MAXGROESSE_IN_MB: process.env['IMPORT_FILE_MAXGROESSE_IN_MB']
+            ? parseInt(process.env['IMPORT_FILE_MAXGROESSE_IN_MB'])
```

Comment:
Wenn das parseInt fehlschlägt weil z.B. ein String reingegeben wurde der in kein int convertiert werden kann wäre ein explizites Fallback zu undefined sinnvoll.

Reply:
Es ist auch schon so 
![{A536DFD2-FFD6-4610-87C6-881CE0D5B1D0}](https://github.com/user-attachments/assets/ad444c1f-99d0-4923-990a-ce49cc280503)

Reply:
Wenn z.B IMPORT_FILE_MAXGROESSE_IN_MB als "23,9A" reingegeben wird, wird parseInt einen Fehler werfen und nicht auf "undefined" zurückfallen. Auf undefined fällt es nur zurück wenn die Condition vor dem "?" false ergibt, nicht aber im Fehlerfall des parsens....